From 40d5cf34f4411e963f0c7a3474cce00d2c62227d Mon Sep 17 00:00:00 2001
From: shangxiudong <shangxiudong@uniontech.com>
Date: Mon, 11 May 2020 16:53:43 +0800
Subject: [PATCH 10/29] Type:fix 19882

Description:When entering Chinese, right-click the input method icon to switch to traditional Chinese, but the actual output is still simplified, and the input method icon on the taskbar is automatically cut into the English keyboard.

RootCause:Null

Solution: I simply modified the user-config-value among data/config.desc to let windows share the same fcitx's global status.
---
 data/config.desc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/config.desc b/data/config.desc
index 03b04e9..d711f7e 100644
--- a/data/config.desc
+++ b/data/config.desc
@@ -164,7 +164,7 @@ Advance=True
 
 [Program/ShareStateAmongWindow]
 Type=Enum
-DefaultValue=No
+DefaultValue=All
 EnumCount=3
 Enum0=No
 Enum1=All
-- 
2.20.1

